daemon/api_notices.go

```diff
@@ -31,24 +31,26 @@ import (
 )
 
 var noticeReadInterfaces = map[state.NoticeType][]string{
-	state.ChangeUpdateNotice:   {"snap-refresh-observe"},
-	state.RefreshInhibitNotice: {"snap-refresh-observe"},
-	state.SnapRunInhibitNotice: {"snap-refresh-observe"},
+	state.ChangeUpdateNotice:       {"snap-refresh-observe"},
+	state.RefreshInhibitNotice:     {"snap-refresh-observe"},
+	state.SnapRunInhibitNotice:     {"snap-refresh-observe"},
+	state.RequestsPromptNotice:     {"snap-prompting-control"},
+	state.RequestsRuleUpdateNotice: {"snap-prompting-control"},
 }
 
 var (
 	noticesCmd = &Command{
 		Path:        "/v2/notices",
 		GET:         getNotices,
 		POST:        postNotices,
-		ReadAccess:  interfaceOpenAccess{Interfaces: []string{"snap-refresh-observe"}},
+		ReadAccess:  interfaceOpenAccess{Interfaces: []string{"snap-refresh-observe", "snap-prompting-control"}},
 		WriteAccess: openAccess{},
```

Comment:
Curious, why is write access more open than read access?

Reply:
the names aren't great perhaps, openAcces{} is open for all requests but on the system socket only, while interfaceOpenAccess{} allows snap-snap socket and checks the plugged interfaces